.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -1,5 +1,5 @@
-name: 🐞 Bug
-description: Report a bug or an issue you've found within the dbt package
+name: 🐞 Bug (non-SQL-translation issues only)
+description: Report a bug or an issue you've found within the dbt package. For SQL translation issues, please contact Fivetran Support.
```

Comment:
```suggestion
description: Report a bug or an issue you've found within the dbt package. For SQL translation issues, please contact Fivetran [Support](https://support.fivetran.com/).
```

Reply:
just in case people forget the link was on the previous page

Reply:
Good callout. Thank you and updated!